src/controllers/auth.controllers.js

```diff
@@ -0,0 +1,111 @@
+import { generateToken} from '../utils/authentication'
+const { validationResult } = require('express-validator');
+const bcrypt = require('bcryptjs');
+const User = require('../models/user.model');
+
+
```

Comment:
please refactor(organize) the functions so it can be more readable,

Reply:
done